### DISCUSSION TITLE
Solana error on metaplex when trying to make auction

### DISCUSSION THREAD

**Discussion by orkaclimatesolutions** [metaplex-foundation/metaplex#2130] [Q&A]
So I’m working on a metaplex project and when I try to launch an auction I’m getting a few errors and when I look for answers I’m not finding anything that makes much sense to me and what could be causing this. (note: I'm pretty new to all of this)

Also search the rust documents and didn’t have luck.

But one error is

#1 Metaplex NFT Auction instruction

Program log: + Processing CreateAuction Program log: Transfer 141190560 lamports to the new account Invoking System Program Program returned success Program log: Allocate space for the account Invoking System Program Program returned success Account data size realloc limited to 10240 in inner instructions Program Metaplex NFT Auction consumed 9991 of 200000 compute units Program returned error: Failed to reallocate account data

Another is

#1 System Program instruction

Program returned success #2 Token Program instruction Program log: Instruction: InitializeAccount Program Token Program consumed 3295 of 200000 compute units Program returned success #3 Metaplex Program instruction Program log: Instruction: Init Auction Manager V2 Program log: This safety deposit box is not listed as a prize in this auction manager! Program Metaplex Program consumed 5430 of 200000 compute units Program returned error: custom program error: 0x1b

Another is

Raw #1 System Program instruction

Program returned success #2 Token Program instruction Program log: Instruction: InitializeAccount Program Token Program consumed 3295 of 200000 compute units Program returned success #3 Metaplex Program instruction Program log: Instruction: Init Auction Manager V2 Program log: This safety deposit box is not listed as a prize in this auction manager! Program Metaplex Program consumed 5430 of 200000 compute units Program returned error: custom program error: 0x1b

Another is

Raw #1 System Program instruction

Program returned success #2 Token Program instruction Program log: Instruction: InitializeAccount Program Token Program consumed 3295 of 200000 compute units Program returned success #3 Metaplex Program instruction Program log: Instruction: Init Auction Manager V2 Program log: This safety deposit box is not listed as a prize in this auction manager! Program Metaplex Program consumed 5430 of 200000 compute units Program returned error: custom program error: 0x1b

Another is

#1 Metaplex NFT Auction instruction

Program log: + Processing CreateAuction Program log: Transfer 141190560 lamports to the new account Invoking System Program Program returned success Program log: Allocate space for the account Invoking System Program Program returned success Account data size realloc limited to 10240 in inner instructions Program Metaplex NFT Auction consumed 9991 of 200000 compute units Program returned error: Failed to reallocate account data

Another is

#1 System Program instruction

Program returned success #2 Token Program instruction Program log: Instruction: InitializeAccount Program Token Program consumed 3295 of 200000 compute units Program returned success #3 Metaplex Program instruction Program log: Instruction: Init Auction Manager V2 Program log: This safety deposit box is not listed as a prize in this auction manager! Program Metaplex Program consumed 5430 of 200000 compute units Program returned error: custom program error: 0x1b

You can see my solscan info too below

Appreciate your help and thank you in advance

https://solscan.io/account/6US2oFFpyidDHTGh1rbBm1AvbHU8ex2aRk1rNsZNqNRj

**Comment by Bunnybits** [comment]
Hey Brother you have a metaplex code file can you send me because the here is the metaplex file is empty please
